apps/web/components/molecules/tabs/tabs.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tab } from '@headlessui/react';
+import clsx from 'clsx';
+import { tabs } from 'molecules/tabs/tabsContent';
```

Comment:
Make tabs a prop and pass it via a prop?
This makes this component strictly bound to this right now. Not reusable